javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -83,7 +87,9 @@ private AnnotationUtils() {
    *
    * @param annotation the annotation whose name to return
    * @return the fully-qualified name of an annotation as a String
+   * @deprecated use {@link annotationName} and {@link java.lang.String#intern}
    */
+  @Deprecated // 2024-10-05
```

Comment:
This method is faster than calling  `annotationName(...).intern()` on `AnnotationBuilder.CheckerFrameworkAnnotationMirror` objects because it doesn't call `intern()` needlessly on the annotation names.  Why do you think it should be deprecated?